src/distro/distro.py

```diff
@@ -898,6 +898,15 @@ def version(self, pretty: bool = False, best: bool = False) -> str:
         if self.uname_attr("id").startswith("aix"):
             # On AIX platforms, prefer oslevel command output.
             versions.insert(0, self.oslevel_info())
+        elif self.id() == "debian" or "debian" in self.like().split():
+            # On Debian-like, add debian_version file content to candidates list.
+            try:
+                with open(
+                    os.path.join(self.etc_dir, "debian_version"), encoding="ascii"
```

Comment:
This additional read^^ may interfere with #328, i.e. one of the two will need a rebase and additional changes after adjustments.

Reply:
Definitely ! If we merge this beforehand, I'll perform a rebase for #328 :angel: